src/libraries/System.Reflection.MetadataLoadContext/tests/ILLink.Descriptors.xml

```diff
@@ -1,3 +1,7 @@
 <linker>
```

Comment:
Should this be conditionally included when EnableAggressiveTrimming is true always then?  (and renamed to indicate that?)

Reply:
We probably don't want that on net6, right?

Reply:
> Should this be conditionally included when EnableAggressiveTrimming is true always then? (and renamed to indicate that?)

It's not EAT specific. I can reproduce it with `/p:PublishTrimmed=true`:

```
/Users/radical/dev/r4/src/mono/wasm/build/WasmApp.targets(336,5): error : FilesToIncludeInFileSystem file '/Users/radical/dev/r4/artifacts/bin/System.Reflection.MetadataLoadContext.Tests/net6.0-Release/browser-wasm/publish/mscorlib.dll' not found [/Users/radical/dev/r4/src/libraries/System.Reflection.MetadataLoadContext/tests/System.Reflection.MetadataLoadContext.Tests.csproj]
```

`mscorlib.dll` is getting trimmed out completely, but some tests want to load that assembly explicitly.

Reply:
> We probably don't want that on net6, right?

We do. IIUC, in net6 it just contains type-forwards.

Reply:
That sounds like our targets are wrong then, look at the csproj

Reply:
https://github.com/dotnet/runtime/blob/030dde1d5867897d6c9a69d0f142d3d3a80cb9a5/src/libraries/System.Reflection.MetadataLoadContext/tests/System.Reflection.MetadataLoadContext.Tests.csproj#L78-L82 should probably just be *.dll

Reply:
@radical tells me it it specifically loads mscorlib, in which case lets update the comment to explain that, not use the error from the build

Reply:
This [test](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Reflection.MetadataLoadContext/tests/src/Tests/MetadataLoadContext/PathAssemblyResolver.cs#L334-L339) is loading `mscorlib.dll` by name, IIUC.

And we don't want to include all the assemblies from `publish/` in the wasm vfs.

Reply:
> This [test](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Reflection.MetadataLoadContext/tests/src/Tests/MetadataLoadContext/PathAssemblyResolver.cs#L334-L339) is loading `mscorlib.dll` by name, IIUC.
> 
> And we don't want to include all the assemblies from `publish/` in the wasm vfs.

test: `System.Reflection.Tests.MetadataLoadContextTests.RelocatableAssembly`